frontend/src/AlertConfiguration/AlertConfigurationComponents/ConfigForm.js

```diff
@@ -86,7 +79,7 @@ export default function ConfigForm({ addConfig }) {
             <MenuItem value="">
               <em>None</em>
             </MenuItem>
-            { dataOptions }
+            { relatedDimensions.map((rd) => <MenuItem value={rd}>{rd}</MenuItem>) }
```

Comment:
should be dimensions instead of relatedDimensions?

Reply:
+1